addons/mrp/static/src/components/wo_list_view_dropdown/wo_list_view_dropdown.js

```diff
@@ -71,9 +80,20 @@ export class MOListViewDropdown extends Component {
             user.context
         );
     }
+
+    async callOrm(functionName){
+        let ids = this.props.record.model.root.selection?.map((element) => element.evalContext.id);
+        // if no records selected, take the current clicked one
+        if (!ids || (ids && ids.length == 0)) {
```

Comment:
if `!ids` is false, `ids` is then true
```suggestion
        if (!ids || ids.length == 0) {
```